web/testing/empty-row/empty_row-1.0.js

```diff
@@ -0,0 +1,521 @@
+KeymanWeb.KR(new Keyboard_empty_row());
+
+function Keyboard_empty_row() {
+    this.KI = "Keyboard_empty_row";
+    this.KN = "empty row";
+    this.KMINVER = "9.0";
+    this.KV = null;
+    this.KDU = 0;
+    this.KH = '';
+    this.KM = 0;
+    this.KBVER = "1.0";
+    this.KMBM = 0x0000;
+    this.KVKL = {
+        "phone": {
+            "font": "Tahoma",
+            "displayUnderlying": false,
+            "layer": [{
+                "id": "default",
+                "row": [{
+                    "id": "1"
+                }, {
```

Comment:
row 1 is a blank row on the default layer.
Also ll. 214, (row 2 on shift layer) and
ll. 423 (row 3 on numeric layer)

Reply:
Per @jahorton , I've changed row 1 to to test array length == 0
```
"row": [{
  "id": "1",
  "key": []
}, {
...
```